src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -278,21 +281,28 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 
     void MainPage::_Navigate(hstring clickedItemTag)
     {
+        _profileViewModelChangedRevoker.revoke();
+        _breadcrumbs.Clear();

```

Comment:
These should go into a `PreNavigateHelper` since this happens in both `_Navigate` functions